### PR TITLE
feat: update challenge and prototype

### PR DESCRIPTION
**Pull request by jalalazimi**
- Changed Challenge to understand better the participant's mastery of the React and Javascript
- I updated the Prototype, but you can't see it in the Readme file. Once we merge it in master, it will be visible.

prototype

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,39 @@
 # Frontend Challenge
 
-The purpose of this challenge is to evaluate your knowledge in the domain of front-end development. We need you to build a Single Page Application with JavaScript frameworks and libraries and, if you deem necessary, data stream libraries of your choice. In addition, we would like you to write some tests. We are looking for readable, easy-to-understand code that follows best practices and design patterns.
+The purpose of this challenge is to evaluate your knowledge in the domain of front-end development. We were hoping you could build an Entities search application with JavaScript frameworks and libraries using AYLIEN News API. Besides, we would like you to write some tests. We are looking for a readable, easy-to-understand code that follows best practices and design patterns.
```
Comment (marcinaylien) [inline README.md:3]: do we really hope? we rather expect/want it from the candidate :)

File: README.md [hunk 2]
```diff
@@ -1,26 +1,39 @@
...
 
 ## The challenge
-We need you to develop Client (UI) and Server architecture which will allow the user to browse the JSON files inside the server folder.
+We need you to develop a Client (UI) architecture that allowing the user to search among entities and see related stories at the bottom of the search bar.
```
Comment (marcinaylien) [inline README.md:6]: that allowing -> that allows

File: README.md [hunk 3]
```diff
@@ -1,26 +1,39 @@
...
-## Terminology
+### Terminology
+- An entity is a thing with a distinct and independent existence.
 - A story contains information about a news article enriched with NLP metadata.
-- Clusters are collections of related stories which share common details.
-- Each cluster has a single representative story to showcase the type of stories it contains.
-**For this test, you will need to provide a way to list the clusters and then display representative stories for each cluster.**
+
+### Description
+For this project, you will need to provide a search bar that helps users search via entities using the `/autocomplete` endpoint and display related stories using the `/stories` endpoint.
```
Comment (marcinaylien) [inline README.md:13]: you will need to provide -> you will provide

Comment (marcinaylien) [reply README.md:13]: that helps users search -> that allows users to search

File: README.md [hunk 4]
```diff
@@ -1,26 +1,39 @@
...
+
+We :heart: React and created a boilerplate project using [Create React App](https://create-react-app.dev/). However, you are free to use any frameworks and libraries you choose. The prototype is just a suggestion — you have complete creative freedom!
```
Comment (marcinaylien) [inline README.md:15]: and created a -> and we already created a

File: README.md [hunk 5]
```diff
@@ -1,26 +1,39 @@
...
 
 <p align="center">
   <img src="https://github.com/AYLIEN/frontend-challenge/blob/master/prototype.png?raw=true">
 </p>
 
-For the client, we :heart: React and created a boilerplate project using [Create React App](https://create-react-app.dev/) However, you are free to use any frameworks and libraries you choose. The prototype is just a suggestion — you have complete creative freedom!
+## API
+For using API, Please register for the News API and follow the following documentations:
```
Comment (marcinaylien) [inline README.md:22]: For using API, Please register -> In order to access API you have to register

File: README.md (hunk 2 above)
Comment (shanethmoore) [reply README.md]:
```suggestion
You are tasked with developing a Client (UI) architecture that will allow the user to search among entities and display stories related to these entities at the bottom of the search bar.
```

File: README.md (hunk 4 above)
Comment (shanethmoore) [reply README.md]:
```suggestion
We :heart: React and have already created a boilerplate project using [Create React App](https://create-react-app.dev/). However, you are free to use any frameworks and libraries you choose. The prototype is just a suggestion — you have complete creative freedom!
```

File: README.md [hunk 6]
```diff
@@ -1,26 +1,39 @@
...
+## Notes
+We would like you to identify the required tasks and provide a time estimation for these tasks before you start working on them. If you can split these tasks into smaller items, please also share those. It’s up to you how much time you invest, but keep in mind that we expect a meaningful contribution and also that it should be possible to review your solution in a reasonable amount of time, so please provide instructions on how to get up and running with your solution and test it. Your solution is expected as a zip file, but please also include your .git directory — we are interested in your development steps, not just the end result.
```
Comment (shanethmoore) [inline README.md:35]:
```suggestion
Before starting the challenge, we would also like you to identify the required steps and provide a time estimate for these. If you can split these tasks into smaller items, please also share these. It’s up to you how much time you invest, but keep in mind that we expect a meaningful contribution and also that it should be possible to review your solution in a reasonable amount of time, so please provide instructions on how to get up and running with your solution and test it. Your solution is expected as a zip file, but please also include your .git directory — we are interested in your development steps, not just the end result.
```

**Review by marcinaylien — APPROVED** [review]

**Pull request merged**